client/src/components/saved.js

```diff
@@ -1,8 +1,10 @@
 import {useState, useEffect} from 'react';
+import CandidateCard from './candidate-card';
```

Comment:
Your code looks good. 
Perhaps provide some comments about what your code is doing, similar to how you have comments on your server that descripe what your routes / tables are doing.